### ISSUE TITLE
Wox tray icon does not exit Wox

### ISSUE THREAD

**Issue by scythk** [Wox-launcher/Wox#1752]
1. Plugins not response：
Clipboard History; Currency Converter; Youdao Translator
Nothing happens after input the keywords

2. Action keywords is null and therefore cannot be clicked and cannot be modified

3. Cannot open wox after exit from tray, unless shutdown the process

**Comment by lances101** [comment]
1. Clipboard History seems to be working as of Wox 1.3.424. Try updating or reinstalling the plugin, and checking in Wox's settings that it is enabled and a keyword is set. 
2. Currency Converter uses Python. Make sure your python directory is set correctly in the settings window.
3. **Confirmed**. Wox is not closed when clicking the corresponding option in the tray icon

What version of Wox are you using? Launch Wox and try calling each of the plugins that supposedly does not work. After that upload the latest log file.

You can find it in `%appdata%\Wox\Logs\<version_folder>\<date>.txt`

For issues you may report down the line, please create a new issue

**Comment by lances101** [comment]
@scythk I need to know what plugins are you using. This seems to be caused by some specific plugins but I want to fix the cause. 

So far one of the culprit plugins seems to be Switcheroo.

**Comment by glenviewjeff** [comment]
I'm having this problem too; I disabled all of my plugins and it Wox still hangs. I wonder if I need to delete all of the plugins rather than just disable them? This is frustrating; I really want to use Wox, but something's misbehaving.

**Comment by glenviewjeff** [comment]
I deleted %APPDATA%\Wox and now it exits; not sure what the culprit was.

**Comment by bao-qian** [comment]
Please try new release https://github.com/Wox-launcher/Wox/releases
If problem still exist, try reset by delete `%APPDATA%Wox` directory.
Reopen if above method won't help.
